lib/src/themes/common_themes.dart

```diff
@@ -39,7 +39,7 @@ AppBarTheme _createAppBarTheme(ColorScheme colorScheme) {
 
 InputDecorationTheme _createInputDecorationTheme(ColorScheme colorScheme) {
   final radius = BorderRadius.circular(kYaruButtonRadius);
-  const width = 1.0;
+  const width = kYaruBorderWidth;
```

Comment:
The constant name here could be misleading. Where else do we use it? Maybe just create a new variable because borders are not that thick elsewhere